packages/azure/changelog.yml

```
@@ -1,3 +1,8 @@
+- version: "1.11.5"
```

Lets increment the minor for enhancement.

```suggestion
- version: "1.12.0"
```